packages/server/modules/core/services/users.js

```diff
@@ -106,7 +106,19 @@ module.exports = {
     delete user.passwordDigest
     delete user.password
     delete user.email
-    await Users( ).where( { id: id } ).update( user )
+
```

Comment:
The user_update activity is already saved from the graphql entrypoint:  https://github.com/specklesystems/speckle-server/blob/main/packages/server/modules/core/graph/resolvers/users.js#L89

Reply:
I see now i was calling the wrong method in my test which is why I added this.  Removed and changed the spec